File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnFieldSetOfText.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnThemePickMultiVariant;
import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldSetOfText extends DefnFieldEditable
{
  @Nullable
  public String[] defaultValue;

  @Nullable
  public MetaIdField defaultValueFieldId;

  @Nullable
  public MetaIdField optionFieldId;

  @Nullable
  public DefnStudioMapOfDtoOption optionMap;

  @Nullable
  public DefnStudioMapOfOptionPermission optionPermissionMap;

  @Nullable
  public Long pageSize;

  @Nullable
  public DefnDtoPluginApi pluginApi;

  @Nullable
  public MetaIdField pluginErrorFieldId;

  @Nullable
  public EnumDefnThemePickMultiVariant showAs;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldSetOfText.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnThemePickMultiVariant
import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldSetOfText : DefnFieldEditable {
    val defaultValue: List<String>?
    val defaultValueFieldId: MetaIdField?
    val optionFieldId: MetaIdField?
    val optionMap: DefnStudioMapOfDtoOption?
    val optionPermissionMap: DefnStudioMapOfOptionPermission?
    val pageSize: Long?
    val pluginApi: DefnDtoPluginApi?
    val pluginErrorFieldId: MetaIdField?
    val showAs: EnumDefnThemePickMultiVariant?
}
